src/react-components/src/components/03-organisms/EditionsList/EditionsList.tsx

```diff
@@ -23,7 +23,8 @@ export default function SearchResultsList(props: SearchResultsListProps) {
           editionHeadingElement={edition.editionYearHeading}
           editionInfo={[edition.publisherAndLocation, edition.language, edition.license]}
           readOnlineLink={edition.readOnlineLink}
-          downloadLink={edition.downloadLink} />
+          downloadLink={edition.downloadLink}
+          noLinkElement={edition.noLinkElement} />
```

Comment:
Curious, where does this get attribute get defined for an edition?

Reply:
Is this specific to ResearchNow?

Reply:
It's real tricky because the original designs are made to be specific to ResearchNow.  "downloadLink" and "readOnlineLink" are also specific to ResearchNow - I wonder if this right-rail really should be a generic span.  
So `noLinkElement` gets defined in the consumer (ResearchNow) ... but @helenvholmes and I have had a conversation about EditionCard, we really need to think through this whole thing more deeply, from a more fundamental level.

Reply:
Yeah, makes sense. From a high-level, does this mean that the server returns a "noLinkElement" value which (I'm assuming) a user defines? And if it's not set or there's no value from the server, the default is `"Unavailable to read online"`? Initially, I was thinking it could use the word `default` in the variable name somewhere, but it seemed there was more to this than I knew about.

Reply:
Yeah, that's what it is from a high level.  `noLinkElement` defaults `Unavailable to read online` but isn't always going to be that.